src/benchmark.cpp

```diff
@@ -486,7 +486,7 @@ BenchmarkSetup setup_benchmark(std::istream& is) {
         int ply = 1;
         for (int i = 0; i < static_cast<int>(game.size()); ++i)
         {
-            const float correctedTime = getCorrectedTime(ply);
+            const float correctedTime = float(getCorrectedTime(ply));
```

Comment:
Honestly, we should just be using `double` throughout here... maybe have `totalTime`, `correctedTime` and `timeScaleFactor` be doubles?

Reply:
I have no strong opinion.  I'm just making the least amount of change possible to fix the warnings in this PR.